### ISSUE TITLE
check for coordinate type in  *2pgtraj

### ISSUE THREAD

**Issue by balazsdukai** [mablab/rpostgisLT#35]
If the coordinates of an ltraj are character and not numeric, ltraj2pgtraj fails. The error msg is not too informative for a non-postgis user.

```
RS-DBI driver: (could not Retrieve the result : ERROR:  function st_makepoint(text, text) does not exist
LINE 31:         ST_SetSRID(ST_MakePoint(x, y), srid) AS geom,
                            ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
). Rolling back transaction
```

**Comment by basille** [comment]
`ltraj` accept coordinates as characters??? Seems like a bug from `adehabitatLT`… (does not prevent from checking it here though!)

**Comment by balazsdukai** [comment]
well, it does give some warnings but the ltraj is created nevertheless, so i think its best to check, because it breaks ltraj2pgtraj

**Comment by basille** [comment]
Yes, definitely. This said, I don't see a good reason for `ltraj` not to enforce it in the first place. Would you mind opening an issue on `adehabitaLT` (maybe it's on purpose, but I really don't see why…).
